includes/class.llms.post-types.php

```diff
@@ -1091,15 +1134,15 @@ public static function register_taxonomies() {
 				'show_admin_column' => true,
 				'show_ui'           => true,
 				'rewrite'           => array(
-					'slug'         => _x( 'course-category', 'slug', 'lifterlms' ),
+					'slug'         => _x( 'r-category', 'slug', 'lifterlms' ),
```

Comment:
is this a typo or what?

Reply:
Damn yea. My 'r' key has been really sticky the past few days.